zscroll

```diff
@@ -163,6 +163,34 @@ parser.add_argument(
          """,
 )
 
+parser.add_argument(
+    '-i',
+    '--insecure-match',
```

Comment:
It would be better if the name clearly described what the flag did. It would be more explicit to name this `-e` / `--eval-match-command-in-shell` or to combine both into `--eval-in-shell`. I'd prefer having just one argument. In practice, I don't think injection will be a problem, and quoting will prevent it. Add a link to the [python docs on shell=True](https://docs.python.org/3/library/subprocess.html#security-considerations) in the manpage and mention in both the help text and manpage that commands should be properly quoted to prevent injection. The help text can just say that command should be properly quoted and to "see the manpage for more details".